backend/spec/system/admin/survey_crud_spec.rb

```diff
@@ -117,6 +131,14 @@
         it 'does not see surveys that not belongs to him' do
           expect(page).not_to have_text(survey2.name)
         end
+
+        it 'see icon_url field that belongs to him' do
+          expect(page).to have_text(survey1.icon_url)
+        end
+
+        it 'does not icon_url field that not belongs to him' do
+          expect(page).not_to have_text(survey2.icon_url)
```

Comment:
Maybe you can add the verb `see` to the description of the test: 
```
it 'does not see icon_url field that not belongs to him' do
```

Reply:
Thanks for the review @PolinaStamenova, I'll add it.